agent-connection/src/main/kotlin/com/luxoft/blockchainlab/corda/hyperledger/indy/AgentConnection.kt

```diff
@@ -214,16 +265,16 @@ class AgentWebSocketClient(serverUri: URI) : WebSocketClient(serverUri) {
     }
 }
 
-data class WalletConnect(val name: String, val passphrase: String, @JsonProperty("@type") val type: String = CONNECT)
+data class WalletConnect(val name: String, val passphrase: String, @JsonProperty("@type") val type: String = CONNECT, val id: String? = null)
```

Comment:
Why do we need Id?
And what commit of agent client you were using? 
It has changed from my version dramatically in latest commits.

Reply:
The id is required in certain messages by the pythonic indy-agent.
I used exactly the last commit in the branch I am sending a PR to. Does this make sense?

Reply:
Ah, my bad, i was talking about agent service implementation.